controllers/machine_controller_test.go

```diff
@@ -318,15 +323,15 @@ func TestMachineFinalizer(t *testing.T) {
 	bootstrapData := "some valid data"
 	clusterCorrectMeta := &clusterv1.Cluster{
 		ObjectMeta: metav1.ObjectMeta{
-			Namespace: "default",
 			Name:      "valid-cluster",
+			Namespace: metav1.NamespaceDefault,
 		},
 	}
 
 	machineValidCluster := &clusterv1.Machine{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "machine1",
-			Namespace: "default",
+			Namespace: metav1.NamespaceDefault,
```

Comment:
Should we use a random generated namespace instead?

Reply:
`fake.Client` is used in this test.